### PR TITLE
Fixed docstrings for graph relabeling and sublattice mapping functionality missing from reference documentation.

### PR DESCRIPTION
**Pull request by qwriter**
Also, edited those docstrings.

This is the fix for issue #234.

### CODE REVIEW

File: dwave_networkx/generators/chimera.py [hunk 1]
```diff
@@ -545,7 +545,7 @@ def __missing__(self, key):
 _chimera_coordinates_cache = __chimera_coordinates_cache_dict()
 
 def linear_to_chimera(r, m, n=None, t=None):
```
Comment (JoelPasvolsky) [inline dwave_networkx/generators/chimera.py:547]: Is this function not documented anywhere?

File: dwave_networkx/generators/chimera.py [hunk 2]
```diff
@@ -653,53 +653,55 @@ def mapping(q):
...
-    integer or coordinate labels.
-    
-    Academic note: the full group of isomorphisms of a Chimera graph includes 
-    mappings which permute tile indices on a per-row and per-column basis, in
-    addition to reflections and rotations of the grid of unit cells where 
-    rotations by 90 and 270 degrees induce a change in orientation.  The full
-    set of sublattice mappings would take those isomorphisms into account; we do
-    not undertake that complexity here.
-    
+    """Yield mappings from a Chimera graph into a larger Chimera graph.
```
Comment (JoelPasvolsky) [inline dwave_networkx/generators/chimera.py:656]: Same comment about internal consistency

File: dwave_networkx/generators/chimera.py [hunk 3]
```diff
@@ -653,53 +653,55 @@ def mapping(q):
...
+
     Parameters
     ----------
         source : NetworkX Graph
-            The Chimera graph that nodes are input from
+            The Chimera graph that nodes are input from.
         target : NetworkX Graph
-            The Chimera graph that nodes are input from
+            The Chimera graph that nodes are output to.
```
Comment (JoelPasvolsky) [inline dwave_networkx/generators/chimera.py:663]: Nice catch!

File: dwave_networkx/generators/chimera.py [hunk 4]
```diff
@@ -653,53 +653,55 @@ def mapping(q):
...
-            A function from nodes of the source graph, to nodes of the target
-            graph.  The offset used to generate this mapping is stored in
-            ``mapping.offset`` -- these can be collected and passed into 
-            ``offset_list`` in a later session.
+            A function from the nodes of the source graph to the nodes 
+            of the target graph.  The offset used to generate this mapping 
+            is stored in ``mapping.offset``, which can be collected and passed 
+            into ``offset_list`` in a later session.
+
+    Notes
```
Comment (JoelPasvolsky) [inline dwave_networkx/generators/chimera.py:677]: I like the idea of using a different section to structure long descriptive content but these docstrings are now inconsistent with the rest of the documentation: the majority of docstrings use the Notes section for minor comments; see for example [maximum_independent_set](https://docs.ocean.dwavesys.com/en/stable/docs_dnx/reference/algorithms/generated/dwave_networkx.maximum_independent_set.html)

**Review by JoelPasvolsky — CHANGES_REQUESTED** [review]
A couple of general comments: 

1. Please make all PRs on a new branch, not on ``main``
2. I made some comments on the first docstrings that apply to the rest as well, so I did not repeat them for all: these are about the use of the descriptive mode versus command for the first line, the use of back ticks for parameter references, and the Notes sections.

File: dwave_networkx/generators/chimera.py (hunk 1 above)
Comment (qwriter) [reply dwave_networkx/generators/chimera.py:547]: Could you elaborate? `linear_to_chimera` does have a docstring.

Comment (JoelPasvolsky) [reply dwave_networkx/generators/chimera.py:547]: There's a ``linear_to_chimera``  method of ``chimera_coordinates`` defined on line 406 and a function ``linear_to_chimera`` defined on line 547, but I see only the first documented on line 40 of ``utilities.rst``

**Comment by arcondello** [comment]
Closing as redundant to https://github.com/dwavesystems/dwave-networkx/pull/238

**Pull request closed without merging**